src/calendar/export/CalendarImporter.ts

```diff
@@ -92,6 +72,63 @@ export function serializeCalendar(
 	return wrapIntoCalendar(versionNumber, "PUBLISH", flat(events.map(({ event, alarms }) => serializeEvent(event, alarms, now, zone))))
 }
 
+//
+// end of the public interface for calendar invites/import/export, everything below this is exported for testing.
+//
+
+/** importer internals exported for testing */
+export async function parseCalendarStringData(value: string, zone: string): Promise<ParsedCalendarData> {
```

Comment:
did this just become async? i see a lot of changed lines in this file but not lots of changes

Reply:
yes, I was confused about the public interface of this thing and moved stuff around. must have missed that I left the now pointless async in. amended the commit to fix it.